src/tools/illink/src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -19,9 +20,7 @@ public abstract class RequiresAnalyzerBase : DiagnosticAnalyzer
 	{
 		private protected abstract string RequiresAttributeName { get; }
 
-		internal abstract string FeatureName { get; }
-
-		private protected abstract string RequiresAttributeFullyQualifiedName { get; }
+		internal abstract string RequiresAttributeFullyQualifiedName { get; }
```

Comment:
Maybe it makes sense to keep the RequiresAttributeFullQualifiedName and FeatureTypeName separated so it's easier to translate for a generic FeatureSwitch analyzer?